### ISSUE TITLE
accessibility: mobile version is not accessible via orca

### ISSUE THREAD

**Issue by pravi** [diaspora/diaspora#5774]
The display switches to mobile version but orca still reads the desktop version.

**Comment by pravi** [comment]
Actually it is reading mobile only but the hidden menu is read before opening it causing confusion.

**Comment by Flaburgan** [comment]
This should be tested again when https://github.com/diaspora/diaspora/pull/5556 will be merged.

**Comment by svbergerem** [comment]
#6233 has been merged some time ago. @pravi it would be great if you could test this again on the develop branch.

**Comment by svbergerem** [comment]
@pravi v0.6.0.0 has been released. It would be great if you could give this another try.

**Comment by svbergerem** [comment]
@pravi Ping.

**Comment by pravi** [comment]
@svbergerem, I had asked my friends to test it. I'll ping them again.
